doc:  update grub menu to match release 3.2 in GSG (#8354)
projectacrn/acrn-hypervisor: doc/getting-started/getting-started.rst

      cp ./acrn*.deb ./grub*.deb ./*acrn-service-vm*.deb /tmp
      sudo apt install /tmp/acrn*.deb /tmp/grub*.deb /tmp/*acrn-service-vm*.deb

> #. Modify /etc/default/grub to show grub menu:

Reviewer: ```suggestion
#. Modify the GRUB menu display using ``sudo vi /etc/default/grub`` and
   comment out the hidden style and changing the timeout to 5 seconds, as shown: 
```

Author: Have modified in GSG. Thanks!

   .. code-block:: bash

      #GRUB_TIMEOUT_STYLE=hidden
      GRUB_TIMEOUT=5

> 

Reviewer: ```suggestion
   and install the new GRUB menu using::
   
      sudo update-grub
```

Author: Have modified in GSG. Thanks!

   The target system will reboot into the ACRN hypervisor and
   start the Ubuntu Service VM.

> #. Confirm that you see the GRUB menu with the "Ubuntu-ACRN Board Inspector" entry. Select

Reviewer: Is this new GRUB menu behavior checked in on master or the release_3.2 branch?  I don't see a PR on either branch that would make this change?

Author: GRUB menu behavior is checked on release_3.2 branch.

   it and proceed to booting ACRN. (It may be auto-selected, in which case it
   will boot with this option automatically in 5 seconds.)

>    Note: Maybe there are several same items display, you can check them via pressng `e` to

Reviewer: These multiple GRUB entries is a rather embarrassing developer experience problem including the documentation "workaround" to check each one (using the 'e'dit option).  Can there be an engineering solution to remove the extra entries? Do we know why there are duplicates for both the hypervisor and board inspector?  Why is there entries for the board inspector that weren't there for 3.1?  @NanlinXie @junjiemao1

Author: Have added kernel version to GRUB menuentry. please refer to #8357 and #8358 for code change.

       select the item with kernel "5.15.0-53-generic".

   Example grub menu shown as below:

   .. code-block:: console

>       :emphasize-lines: 9

Reviewer: ```suggestion
      :emphasize-lines: 7
```

   The target system will reboot into the ACRN hypervisor and
   start the Ubuntu Service VM.

#. Confirm that you see the GRUB menu with the "Ubuntu-ACRN Board Inspector" entry. Select
   it and proceed to booting ACRN. (It may be auto-selected, in which case it
   will boot with this option automatically in 5 seconds.)

   Note: Maybe there are several same items display, you can check them via pressng `e` to

>        select the item with kernel "5.15.0-53-generic".

Reviewer: I see 5.15.0-56-generic?

Author: When moving the curser to "Ubuntu-ACRN Board Inspector" entry, press the key `e`  then GRUB display will contain kernel information `Linux 5.15.0-53-generic`.
The output shows us that we are running kernel version 5.15.0-53.
Here is what each of these numbers mean:
5 – Kernel version.
15 – Major revision.
0 – Minor revision.
53 – Patch number.

So it's the difference between our kernel versions (both OS are Ubuntu22.04). 
Do you think we should modify the sample file to describe the kernel like this:
  "5.15.*-*-generic"?
Or other opinion? Thanks!

Author: Maybe the kernel version upgraded. Have modified it in GSG. Thanks!
